src/components/common/Content/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+@import 'styles/mixins';
+
+.content {
+    padding: 20px 0;
+    min-height: calc(100vh - 60px - 332px);
```

Comment:
what is 60px?
what is 332px?

probably it's height of header and footer?

If yes - we can create variable in `styles/variables` and use it here